# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/credentialmanagement/CredentialsContainer.h
# Commit: 287bedb227e8
# Full Hash: 287bedb227e89933347633242efe1acd4236ad70
# Author: John Schanck <jschanck@mozilla.com>
# Date: 2025-02-12 09:32:07
# Regressor Bug: 1945969
# File Overlap Count: 1
# Description:
#   Bug 1945969 - part 5: rename WebAuthnManager to WebAuthnHandler. r=keeler
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D237257
# ==============================================================================

diff -r a01c165a2524 -r 287bedb227e8 dom/credentialmanagement/CredentialsContainer.h
--- a/dom/credentialmanagement/CredentialsContainer.h	Wed Feb 12 02:12:58 2025 +0000
+++ b/dom/credentialmanagement/CredentialsContainer.h	Wed Feb 12 02:12:58 2025 +0000
@@ -11,7 +11,7 @@
 
 namespace mozilla::dom {
 
-class WebAuthnManager;
+class WebAuthnHandler;
 
 class CredentialsContainer final : public nsISupports, public nsWrapperCache {
  public:
@@ -22,7 +22,7 @@
 
   nsPIDOMWindowInner* GetParentObject() const { return mParent; }
 
-  already_AddRefed<WebAuthnManager> GetWebAuthnManager();
+  already_AddRefed<WebAuthnHandler> GetWebAuthnHandler();
 
   virtual JSObject* WrapObject(JSContext* aCx,
                                JS::Handle<JSObject*> aGivenProto) override;
@@ -41,10 +41,10 @@
  private:
   ~CredentialsContainer();
 
-  void EnsureWebAuthnManager();
+  void EnsureWebAuthnHandler();
 
   nsCOMPtr<nsPIDOMWindowInner> mParent;
-  RefPtr<WebAuthnManager> mManager;
+  RefPtr<WebAuthnHandler> mWebAuthnHandler;
   bool mActiveIdentityRequest;
 };
 